sbin/umount/umount.c

```diff
@@ -70,24 +80,28 @@ int	 checkname (char *, char **);
 int	 umountfs(struct statfs *sfs);
 void	 usage (void);
 int	 xdr_dir (XDR *, char *);
+int	 md_detach(const char *);
```

Comment:
Since this is only used in umount.c, it should be static... but all the other functions above it should be too, so consistency wins out over standard practice. Just  fyi for next time.